README.md

```diff
@@ -19,7 +19,7 @@ An application template for [RustyHermit](https://github.com/hermitcore/rusty-he
     hermit-sys = "<version>"
     ```
 
-*   Use the exact Rust version required by `hermit-sys` in `rust-toolchain.toml` and make the `rust-src` component available:
+*   Use the exact Rust version [required by `hermit-sys`](https://github.com/hermitcore/rusty-hermit/blob/master/rust-toolchain.toml) in `rust-toolchain.toml` and make the `rust-src` component available:
```

Comment:
> @mkroening Is this actually the best location to find the required toolchain version?

Well, yes. Though, we should link to this repository's `rust-toolchain.toml` instead (rusty-hermit's git contains unreleased updates).

I think, this should work:

```suggestion
*   Use the [exact Rust version] required by `hermit-sys` in `rust-toolchain.toml` and make the `rust-src` component available:

[exact Rust version]: rust-toolchain.toml#L2
```